src/create-or-update-comment.ts

```diff
@@ -120,9 +120,10 @@ function appendSeparatorTo(body: string, separator: string): string {
 
 function truncateBody(body: string) {
   // 65536 characters is the maximum allowed for issue comments.
+  let truncateWarning = "*Comment has been truncated*"
```

Comment:
```suggestion
  let truncateWarning = "...*[Comment body truncated]*"
```

e.g.
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus aliquam felis sed risus accumsan blandit. Integer varius tellus ut sem tempus, eu efficitur ex varius. Nunc vitae imperdiet ante. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Interdum et malesuada fames ac ante ipsum primis in faucibus. Vestibulum et aliquet risus. Nulla et com...*[Comment body truncated]*